src/common/styles.c

```diff
@@ -823,7 +823,19 @@ void dt_styles_apply_to_image(const char *name, const gboolean duplicate, const
     gchar ntag[512] = { 0 };
     g_snprintf(ntag, sizeof(ntag), "darktable|style|%s", name);
     if(dt_tag_new(ntag, &tagid)) dt_tag_attach_from_gui(tagid, newimgid, FALSE, FALSE);
-    if(dt_tag_new("darktable|changed", &tagid)) dt_tag_attach_from_gui(tagid, newimgid, FALSE, FALSE);
+    if(dt_tag_new("darktable|changed", &tagid))
+    {
+      dt_tag_attach_from_gui(tagid, newimgid, FALSE, FALSE);
+
+      /* register time in main.images.change_timestamp */
+      DT_DEBUG_SQLITE3_PREPARE_V2(dt_database_get(darktable.db),
+          "UPDATE main.images SET change_timestamp = ?1 WHERE id=?2 ", -1, &stmt, NULL);
```

Comment:
`SET change_timestamp = STRFTIME('%s', 'now') `

And you avoid an SQL parameter.

Reply:
... And you avoid an SQL parameter.
I don't understand. Its work for me.

Reply:
Sure it works, but using strftime you remove a bind, remove a sys call which could be OS dependent.